resources/serverless/values.yaml

```diff
@@ -82,13 +82,13 @@ images:
   runtimes:
     nodejs12:
       repository: "eu.gcr.io/kyma-project/function-runtime-nodejs12"
-      tag: "5daaf5c9"
```

Comment:
I'm no expert but shouldn't those be bumped? 
eu.gcr.io/kyma-project/function-webhook:f4edbb0a
eu.gcr.io/kyma-project/function-build-init:f4edbb0a
eu.gcr.io/kyma-project/function-controller:f4edbb0a

Reply:
sorry my bad